quiz.html

```diff
@@ -46,7 +46,7 @@ <h2>1 / 5</h2>
 							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Dry">Dry</p></div>
 							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Normal">Normal</p></div>
 							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Sensitive">Sensitive</p></div>
-							<hr color="white" width="80%" size="3">
+							<hr color="black" width="80%" size="3">

```

Comment:
In version 2, I changed the horizontal line to black to match the colour scheme better.